### DISCUSSION TITLE
Moving Camera on a vehicle

### DISCUSSION THREAD

**Discussion by nickpoorman** [blakeblackshear/frigate#2542] [Ask A Question] [answered]
This may be an odd question…

I’m running Frigate and HomeAssistant in my RV. When the RV is moving that means the cameras outside are moving. I would like to disable/pause the detections and anything else that’s going to cause high device usage while the vehicle is in motion but continue saving the recordings from the video stream.

Would sending the disable detections MQTT message accomplish this? Are there other settings that I should disable while the vehicle/ camera is in motion as so that things don’t go haywire?

I have the ability to automate this when the vehicle starts and stops.

**Comment by itkfilelor** [comment, marked as answer]
Yes, mqtt would solve this. ~~Do you already have a method setup in HA to test for vehicle motion?~~
I have certain cameras basically shut off except live until sunset. I turn off recordings, and detection via mqtt. 
What method are you using for motion detection for the RV?

**Reply by nickpoorman** [reply]
Ok, great. Thank you.

I have a Victron power system. The DC/DC charger looks for a spike in voltage to tell it when the vehicle alternator is on so it knows when to pull juice to charge the house batteries. The Victron Cerbo GX system runs node red so I can trigger things based on state changes in the various Victron components.
